docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _discords(T, m, k=1, r_init=None, decay=None, s=None, include=None, finite=False):
</pre><p>Let's call the input parameter <code>r</code> instead of <code>r_init</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Will be changed in next set of commits.</p>